extension/api.js

```diff
@@ -58,14 +58,14 @@
 	})();
 
 	window.gitHubNotifCount = function (cb) {
-		var token = GitHubNotify.settings.get('oauthToken');
+		var token = window.GitHubNotify.settings.get('oauthToken');
```

Comment:
The only reason I did this is for the tests. It does not affect the behavior when running in chrome, but mocha/node need the `window.` prepended.